src/Sulu/Bundle/SearchBundle/Controller/WebsiteSearchController.php

```diff
@@ -109,6 +117,16 @@ public function queryAction(Request $request)
         );
     }
 
+    private function resolveIndexPlaceholders($value)
+    {
+        $resolver = $this->container->get('twig');
```

Comment:
The expression language would match a little bit better, but twig templates are also compiled and cached, so the overhead won't be that different. The problem I have here is the complexity added, you have to know the webspace object to only configure a simple index name.

Please also have a look at [https://github.com/sulu/sulu/pull/2813#discussion_r75621518](my other comment), which includes another suggestion making this obsolete anyway.